### DISCUSSION TITLE
Does wasm micro runtime supports for ARM Cortex A9 @ 600Mhz

### DISCUSSION THREAD

**Discussion by ShubhamGupta577** [bytecodealliance/wasm-micro-runtime#3934] [Q&A]
Hi
As seen on the wasm micro runtime github, there the mentioned architecture are ARM, THUMB (ARMV7 Cortex-M7 and Cortex-A15 are tested), but there is no mention for other cortex chipset.
Can you please confirm me that whether wasm micro runtime support **ARM cortex A9**, as currently the application for which I am working is going to deploy on target which is based on that.

**Comment by TianlongLiang** [comment]
I tested it with Zephyr and Qemu, and WAMR has no problem running on Cortex A9.

Build with following command:
```bash
west build -b qemu_cortex_a9 . -p always -- -DWAMR_BUILD_TARGET=ARM
```

Run with following command:
```bash
qemu-system-arm -M xilinx-zynq-a9 -m 256M -kernel build/zephyr/zephyr.elf -nographic

elaspe time 146
*** Booting Zephyr OS build v4.0.0-850-gf90ce01d4ac5 ***
Hello world!
buf ptr: 0x1458
buf: 1234
elapsed: 8
```
